src/test/groovy/com/google/protobuf/gradle/ProtobufPluginTestHelper.groovy

```diff
@@ -27,24 +26,37 @@ final class ProtobufPluginTestHelper {
     }
   }
 
-  static BuildResult buildAndroidProject(
-      File mainProjectDir,
-      String gradleVersion,
-      String agpVersion,
-      String fullPathTask,
-      String... arguments
+  static GradleRunner getGradleRunner(
+    File projectDir,
+    String gradleVersion,
+    String... arguments
   ) {
-    return getAndroidGradleRunner(mainProjectDir, gradleVersion, agpVersion, fullPathTask, arguments).build()
+    List<String> args = arguments.toList()
+    args.add("--stacktrace")
+
+    return getGradleRunner(projectDir, gradleVersion, args)
+  }
+
+  static GradleRunner getGradleRunner(
```

Comment:
It's surprising that this overload doesn't add `--stacktrace`. Since it is only used within this class, we can either rename it and make it private or have it add `--stacktrace` and remove the `--stacktrace` from getAndroidGradleRunner() and the vararg version of getGradleRunner().